### ISSUE TITLE
Building up the Farming Skill

### ISSUE THREAD

**Issue by Jacqhammer** [StrangeLoopGames/EcoSuggestions#1340]
Farming suggestion.

Currently in version 9.1, farming is a bit useless. Your standard server is going to have maybe 1 or 2 farmers on it. The farmer only fills 1 critical role, research labor for the milling book. Outside of the labor role, there is not much need for farmers. The only meaningful ability the farmer has is seed creation. With that skill the farmer can make a ton of seeds to plant crops, but there is no market of it due to the high respawn rate of crops. Cooks are more likely to go "scavenge" crops then to buy the seeds and grow them. A lot of new players take farming because they think they can grow these massive fields and sell the food. Unfortunately, that's not case. Don't get me wrong, farming is a really cool aspect to the game with the idea of biome crops, growing areas, the tools and NPK's. On paper it looks great, in the game farming gets dumped quick.

If farming was more like engineering, having direct connections to different professions and late game skills, it would have massive value. A great aspect to add would be crop domestication and animal domestication. With crop domestication, farmers could get the ability to produce crop seeds unique to the farming skill. Like domestication of wheat into barley. Then having chefs required to use the domesticated crops in cooking/baking recipes. That alone would make farming an early to late game skill. Also farmers need more "benches" The idea of hoeing and planting crops in the ground is great, but its not practical for a farmer to have 5 different farmers across a world. For a low skill farmer it makes sense, but a high skill farmer should have more control over where crops can grow. Adding in "farm" plots that acted like benches would allow farmers to have more control over crop production. So early game the farmer can build a 5x5 farm plot bench and grow crops regardless of the environment, then late game they can build 20x20 farm plots that need water hookups and power. As a bench the farm plot would be immune to falling tree desire knocking out crops. Also the recipe for crops could be seeds and fertilizers. Currently fertilizers and not used, making it a useless skill. With a "farm" plot bench, it could be built by the farmer skill, use the fertilizer skill to make items and use gathering skill as labor. That way anyone can use the bench, increasing the value of the farmer.

The problem is it would make the steam tractor useless, but it already is based on the lack and need for farming. But that's an easy fix, just add a lumber, mining and road tool attachment for the steam tractor. That would make the steam tractor more valuable to everyone making mid game enhanced for other skills.

The only other addon would be "lamp" oil to the fertilizer skill. Currently every light item besides the blazer requires either oil or tallow to run. Oil only comes mid game and tallow is to valuable for its low burn time. So all the cool house items for lights rarely get made at all due to the heavy need of oil and tallow on foods. Having some type of lamp oil that is made with fertilizers skill, would increase fertilizers value until electric lights came in.

I think with these addons It would increase server retention and add more quality to the items and skills already in the game.

**Comment by danisani1** [comment]
It is a good point, the "farming" skill is really only making seeds. I would swap these farm plots with "greenhouses" (glassworking + or as a tent, less efficient and smaller but with tailoring ressources)

**Comment by sirPattrick** [comment]
A greenhouse Item would be neat.  The ability to adjust the settings to the specific plant you are trying to grow.  For instance a "Corn" setting to make conditions perfect for corn to grow within the area.  maybe powered by a powerplant  (hopefully far enough away to not pollute the crops you are trying to grow)

**Comment by JestinJae** [comment]
I agree that I would love greenhouses.  I feel like farming has a bit more of a role that originally noted in this issue though.  Perhaps the respawn rates have changed?  I have only been in the game since very recently.
